packages/plugin/src/theia-proposed.d.ts

```diff
@@ -168,35 +168,6 @@ export module '@theia/plugin' {
         color?: ThemeColor;
     }
 
-    // #region LogLevel: https://github.com/microsoft/vscode/issues/85992
-
-    /**
-     * The severity level of a log message
-     */
-    export enum LogLevel {
-        Trace = 1,
```

Comment:
It's only "proposed", but we might want to add a breaking change notice in case an adopter is using this API.